exercises/acronym/example.js

```diff
@@ -1 +1 @@
-export const parse = phrase => phrase.match(/[A-Z]+[a-z]*|[a-z]+/g).reduce((acronym, word) => `${acronym}${word[0]}`, '').toUpperCase();
+export const parse = phrase => phrase.replace("'", '').match(/^[A-Z]|(?<=[^A-Z])[A-Z]|\b[a-z]/g).join('').toUpperCase();
```

Comment:
Might be worth breaking this into separate lines for readability.

Reply:
Oh feel free! 

```javascript
export const parse = phrase => phrase
  .replace("'", '')
  .match(/^[A-Z]|(?<=[^A-Z])[A-Z]|\b[a-z]/g)
  .join('')
  .toUpperCase();
```

Reply:
```suggestion
export const parse = phrase => phrase
  .replace("'", '')
  .match(/^[A-Z]|(?<=[^A-Z])[A-Z]|\b[a-z]/g)
  .join('')
  .toUpperCase();
```